lib/functions/compilation/patch/drivers_network.sh

```diff
@@ -505,10 +505,28 @@ driver_rtl8822BS()
 
 }
 
+driver_uwe5622_allwinner()
+{
+	# Unisoc uwe5622 wireless Support
+	if linux-version compare "${version}" ge 4.4 && linux-version compare "${version}" le 6.1 && [[ "$LINUXFAMILY" == sun* ]]; then
```

Comment:
`sun*` -- I see what you're doing here, but who's gonna find this in the future?
(also... LINUXFAMILY is either `sunxi` or `sunxi64` while BOARDFAMILY has more `sun`'s)

Reply:
-> [[ "$LINUXFAMILY" == sunxi* ]] or removing this condition and build always?

Reply:
`sunxi*` will show up in searches for `sunxi` so I'm happy with that